src/ManualSetupPushController.js

```diff
@@ -178,31 +178,36 @@ export default FormController.extend({
         }),
         FormType.View({
           View: View.extend({
-            template: hbs(
-              '\
-                <p class="okta-form-subtitle o-form-explain text-align-c">\
-                  {{instructions}}\
-                </p>\
-              '
-            ),
+            className: 'secret-key-instructions',
+            attributes: { 'data-se': 'secret-key-instructions'},
+            template: hbs`
+            <section aria-live="assertive">
+              <p class="screen-reader-only">{{i18n code="enroll.totp.sharedSecretInstructions.aria.intro" 
+                bundle="login"}}</p>
+              <ol>
+                <li>{{step1Instruction}}</li>
+                <li>{{i18n code="enroll.totp.sharedSecretInstructions.step2" bundle="login"}}</li>
+                <li>{{i18n code="enroll.totp.sharedSecretInstructions.step3" bundle="login" 
+                $1="<strong>$1</strong>" $2="<strong>$2</strong>"}}</li>
+                <li>{{i18n code="enroll.totp.sharedSecretInstructions.step4" bundle="login"}}</li>
+              </ol>
+              <p class="shared-key margin-top-10" tabindex=0 
+                aria-label="{{i18n code="enroll.totp.sharedSecretInstructions.aria.secretKey" bundle="login"
+                arguments="sharedSecretKey"}}">{{sharedSecretKey}}</p>
```

Comment:
Screen readers will likely read aloud the shared secret multiple times - resulting in a poor experience. Consider using only the value of the `sharedSecretKey` as the message here.

Reply:
when we have aria-label it overrides the content in the element. hence we had to append secret key with aria-label